Task 3/system/Coding/src/app/Parameters.c

```diff
@@ -10,39 +10,62 @@
 
 /* INCLUDES ***************************************************************************************/
 #include "Parameters.h"
-#include "Display.h"
 
 /* CONSTANTS **************************************************************************************/
-#define MAX_NUMBER_OF_PARAMETER_SETS 3
+
+#define MAX_NUMBER_OF_PARAMETER_SETS 3u
+#define TEXT_SET1 ("A: NAME SET 1")
+#define TEXT_SET2 ("B: NAME SET 2")
+#define TEXT_SET3 ("C: NAME SET 3")
 /* MACROS *****************************************************************************************/
 
 /* TYPES ******************************************************************************************/
 
 /* PROTOTYPES *************************************************************************************/
 
 /* VARIABLES **************************************************************************************/
-
+/** array of the choosable parameter sets */
 static ParameterSet gParameterSets[MAX_NUMBER_OF_PARAMETER_SETS];
 
-static UInt8 gSelectedParameterSetIndex;
+ParameterSet* pParameters = &gParameterSets[INDEX_SET1];
 
 /* EXTERNAL FUNCTIONS *****************************************************************************/
 
-void Parameters_setParameterSet(void)
+extern void Parameters_setParameterSet(UInt8 index)
 {
-
-
+    pParameters = &gParameterSets[index];
 }
 
-ParameterSet Parameters_getParameterSet(void)
+extern void Parameters_displayParameterSets(void)
 {
-
-
+    Display_clear();
+    Display_gotoxy(0, 0);
+    Display_write(TEXT_SET1, sizeof(TEXT_SET1));
+    Display_gotoxy(0, 1);
+    Display_write(TEXT_SET2, sizeof(TEXT_SET2));
+    Display_gotoxy(0, 2);
+    Display_write(TEXT_SET3, sizeof(TEXT_SET3));
 }
 
-void Parameters_displayParameterSets(void)
+extern void Parameters_initParameterSets(void)
 {
-
-  
+    /** Init of set 1 */
```

Comment:
Inline comments have to only start with /* ... */

if marked with /** at the beginning this will generate documentation in doxygen which is not what we want here.

Also see following.

```suggestion
    /* Init of set 1 */
```